utils/makeWebpackConfig.js

```diff
@@ -13,20 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const path = require('path');
-const webpack = require('webpack');
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-const BundleAnalyzerPlugin = require('webpack-bundle-analyzer')
+const path = require("path");
```

Comment:
Why is your editor converting single quotes to double quotes?  We don't want that.  Do we maybe need an `.editorconfig` file?